lib/app/admin/data-demo/file.js

```diff
@@ -0,0 +1,5 @@
+
+module.exports = function() {
```

Comment:
Should probably name the functions for consistency and ease of reading.

Something like `getSampleFiles`